topics/chip-seq/tutorials/formation_of_super-structures_on_xi/tutorial.md

```diff
@@ -0,0 +1,862 @@
+---
+layout: tutorial_hands_on
+topic_name: chip-seq
+tutorial_name: formation_of_super-structures_on_xi
+---
+
+# Introduction
+{:.no_toc}
+
+Within a cell nucleus, the DNA is tightly-packed and the chromatin is spatially distributed with different levels and scales of organizations.
+
+At the smallest scale, DNA is packaged into units called nucleosomes, made of eight histone proteins. 
+
+At a larger scales than nucleosomes, DNA is forming loops. DNA elements elements that would be otherwise be separated by large distances can interact. The corresponding self-interacting (or self-associating) domains are found in many organisms: they are called Topologically Associating Domains (TADs) in mammalian cells. Mammalian chromosomes are also partitioned into two spatial compartments, labelled "A" and "B", where regions in compartment A tend to interact preferentially with A compartment-associated regions than B compartment-associated ones. Similarly, regions in compartment B tend to associate with other B compartment-associated regions.
+
+In mammals, the X chromosome inactivation (XCI) balances the dosage of X-linked genes between females and males. The genes on the inactive X (Xi) chromosome are not expressed. 
+
+Some changes in the histone modifications could be involved. For example, the H3K4m3 is adding 3 methyl-group of the 4th Lysine in the histone 3 amino-acid. This modification is known to activate the transcription on nearby genes by opening the chromatine. The H3K27me3 on the other hand is inactivating the transcription of the nearby genes:
+
+![Fadloun et al, 2013](../../images/formation_of_super-structures_on_xi/histone_modifications.jpg "Source: Fadloun et al, 2013")
+
+More H3K27me3 and less H3K4m3 on the Xi could explain a lower expression of the genes there. 
+
+It has been also observed that the Xi adopts a distinct conformation without evident compartments or TAD. Cohesins, condensins and CCCTC-binding factor (CTCF) play key roles in chromosomal architectures and TAD formation, other potential cause of the repression of the expression of the genes on Xi.
+
+The structural-maintenance-of-chromosomes hinge domain containing 1 (SMCHD1) has been found enriched on the Xi. It may be the potential actor in the shape of Xi and the change in gene expression there.
+
+[Wang et al. (2018)](https://www.cell.com/cell/fulltext/S0092-8674(18)30584-1) investigates the mechanism by which the SMCHD1 gene shapes the Xi and represses the expression of the genes on Xi in mouse.
+
+Their idea was to identify the differences could be observed between the Xi and activated X chromosome, on both wild-type and SMCHD1 gene knockdown samples to study the SMCHD1 effect. 
```

Comment:
- "Cohesins, condensins and CCCTC-binding factor (CTCF) play key roles in chromosomal architectures and TAD formation, other potential cause of the repression of the expression of the genes on Xi." (feels like an incomplete sentence, better put a "which are" instead of a comma).
- "Their idea was to identify the differences could be ..." (which could be)